Source/Core/Updater/Main.cpp

```diff
@@ -648,7 +658,7 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine
 
   if (!File::IsDirectory(opts.install_base_path))
   {
-    fprintf(log_fp, "Cannot find install base path, or not a directory.\n");
+    FatalError("Cannot find install base path, or not a directory.");
```

Comment:
quick note - one thing i've found really helpful (in C code, please translate mentioned code to C++) is to make a FatalErrorf(const char *fmt, ...) { } instead.
for example, this error could name the exact opts.install_base_path that it's trying to find